Formula/ecl.rb

```diff
@@ -14,11 +14,34 @@ class Ecl < Formula
   end
 
   depends_on "gmp"
+  depends_on "bdw-gc"
+  depends_on "libffi"
 
   def install
-    system "./configure", "--prefix=#{prefix}",
+    gmp = Formula["gmp"]
+    bdwgc = Formula["bdwgc"]
+    libffi = Formula["libffi"]
+
+    # These environment variables are necessary or bdwgc won't be found until the
+    # next upstream release:
+    # https://gitlab.com/embeddable-common-lisp/ecl/issues/346
+    ENV.append "CFLAGS", "-I#{bdwgc.opt_include}"
+    ENV.append "CXXFLAGS", "-I#{bdwgc.opt_include}"
+    ENV.append "LDFLAGS", "-L#{bdwgc.opt_lib}"
+
+    # Parallel-make support is being actively worked on and there is not a PR
+    # ready at this time:
+    # https://gitlab.com/embeddable-common-lisp/ecl/tree/revert-make-split
```

Comment:
Ok, not sure what that means but I assume "this will be fixed in the next upstream release" in which case this comment can be: 

"Parallel-make support is has been merged upstream and will be available in the next release".

Out of interest, @dkochmanski, when do you expect the next release to be?